README.md

```diff
@@ -1,73 +1,63 @@
-targetcli
-=========
+# iscsi-target
 
-Targetcli target iSCSI role which takes care of installing targetcli, enabling it on boot and configuring it based on the provided variable from ansible.
+iSCSI target role that installs and uses targetcli.
 
-NOTE: Modules used by this module were separated into role OndrejHome.targetcli-modules
+## Requirements
```

Comment:
In our case, zfs-storage creates the devices (ZVOLs). This is not included here, as this role is not zfs-specific and can export any device.
You can find the link in the dependencies section where it belongs to.